docs/faq.rst

```diff
@@ -251,6 +251,26 @@ using a Read the Docs :doc:`config-file/index`.
 .. _pip requirements file: https://pip.pypa.io/en/stable/user_guide.html#requirements-files
 
 
+I need to install a package in a environment with pinned versions
+-----------------------------------------------------------------
+
+To ensure proper installation of a python package, the ``pip`` :ref:`install
+method <config-file/v2:python.install>` will automatically upgrade every
+dependency to its most recent version in case they aren't pinned by the package definition. If instead you'd like to pin your
+dependencies outside the package, you can add this line to your requirements or
+environment file (if you are using Conda).
```

Comment:
We use these conventions for wrapping long lines https://docs.readthedocs.io/en/stable/development/docs.html#content